selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/Button.java

```diff
@@ -38,9 +38,9 @@
 
   public final By ELEMENT_SAVE_CLOSE_BUTTON_2 = By.xpath("//*[text()='Save And Close']");
 
-  public final By ELEMENT_OK_BUTTON = By.xpath("//*[text()='OK']");
+  public static final By ELEMENT_OK_BUTTON = By.xpath("//*[text()='OK']");
```

Comment:
Looks like a very generic xpath. There probably can be others OK button in the page in some cases. We should use a more precise xpath IMO.